tasks/bootstrap-deploy-user.yml

```diff
@@ -25,20 +25,15 @@
   authorized_key:
     user: "{{ sb_debian_base_deploy_user }}"
     key:  "{{ item }}"
-  with_file:
-    - keys/deploy_users
-    - keys/ci-{{ sb_debian_base_app_environment }}
+  with_items: "{{ sb_debian_base_deploy_user_authorized_keys }}"
 
-- name: Add SSH keys for GitHub's "Deploy keys"
-  template:
-    src:  "{{ item.src }}"
-    dest: "{{ item.dest }}"
-    mode: "{{ item.mode }}"
+- name: Add deploy user private key
+  copy:
+    dest:  "{{ sb_debian_base_deploy_user_dir }}/.ssh/id_rsa"
+    content: "{{ sb_debian_base_deploy_user_private_key }}"
```

Comment:
@lpaulmp  I'm not sure about defining default values for `sb_debian_base_admin_user_authorized_keys` and `sb_debian_base_deploy_user_authorized_keys` that may lead to potential SSH lockout problems, if you disable SSH password login without setting up some authorized keys first. I would rather raise and error if `sb_debian_base_admin_user_authorized_keys` or `sb_debian_base_deploy_user_authorized_keys` are not defined, I think is a little bit more safe. What do you think? \cc @goetzc